modules/auxiliary/admin/mssql/mssql_enum.rb

```diff
@@ -17,14 +18,18 @@ module to work, valid administrative user credentials must be
         supplied.
       },
       'Author'         => [ 'Carlos Perez <carlos_perez[at]darkoperator.com>' ],
-      'License'        => MSF_LICENSE
+      'License'        => MSF_LICENSE,
+      'SessionTypes'   => %w[MSSQL],
     ))
   end
 
   def run
     print_status("Running MS SQL Server Enumeration...")
+    if (datastore['SESSION'] && session)
```

Comment:
did you run into issues just checking for `session`? I don't think you need to explicitly check `datastore['SESSION']` that should be handled by the call to `session`

Reply:
I'll double check this but I think that makes sense